### PR TITLE
Governance

### LINKED ISSUE #272: Separate the Roadmap + user stories to their own document 📃
### Prerequisites

- [x] I have checked the existing Issues for a duplicate of this ticket
- [x] I have checked the roadmap in the documentation and am convinced this Issue fits the plans for future releases
- [x] This Issue will help us achieve at least one of the existing User Stories for the product (directly or indirectly)

### Expected Behavior

The Roadmap and User Stories should be quick to find for anyone navigating our documentation and easier for us to edit in future, since these will need adapting with each new release.

### Current Behavior

Based on feedback we've gotten the roadmap can be difficult to spot as contributors are not necessarily expecting product info to be in the `CONTRIBUTING.md`

### Any other notes

We might just have a `PRODUCT_ROADMAP.md` and add it to the list of docs linked in the `README.md`

### Assignment

This issue is free for anyone to take

### LINKED ISSUE #274: Add `GOVERNANCE.md` 🛠️
### Expected Behavior

Our documentation needs guidance on different roles can, and are encouraged, to take. All base permissions are set to read but people should know easily what other roles they can take on if they wish to become more involved in the project.

### Current Behavior

`CONTRIBUTING.md` mentions different ways to contribute and we encourage discussions and peer-reviews on issues and prs but the documentation doesn't reflect other roles people can take on.

### Any other notes

_No response_

### Assignment

This issue is free for anyone to take

### PR DESCRIPTION
**Pull request by camelPhonso**
**Closes #274 && Closes #272**  

Set up some guidance on the governance of Kindly.

Outlined four different roles that anyone can take when contributing to Kindly with the expectation on those roles and how to take them on. Also added answers to some questions that I think can be reasonably anticipated.

### Files changed

- `GOVERNANCE.md` - new file.
- `README.md` - cleared directions around the original fac28 repo that are now irrelevant and added a link to `GOVERNANCE.md`
- `PRODUCT_ROADMAP.md` && `CONTRIBUTING.md` - moved the roadmap and user stories out to a new document

### UI changes

None 🙅

### Changes to Documentation

This is a documentation PR. All changes recorded above

# Tests

This PR does not affect the behaviour of the code. No tests required

### CODE REVIEW

File: .github/GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Kindly is maintained through volunteer work - all contributions are welcome and we look to invite new collaborators at all times.
+The roles described below are intended as guidelines for different ways to get involved and different levels of commitment you may choose to take on at any point.
+
+How much you contribute and how long for is entirely at your discretion and collaborators can change their roles in the community in any way at any time that is relevant to them.
+
+There are two simple steps to taking on any role:
+
+- 🔨 **Do** the work involved.  
+  This community is run on willful collaboration, meaning we invite and expect everyone to identify the areas of Kindly they would like to help with and take any reasonable action towards improving the platform.  
+  If you are interested in being a maintainer, find ways in which you can help maintainers with their tasks;
```
Comment (eliazzo) [inline .github/GOVERNANCE.md:12]: is the semi colon intentional here?

File: .github/GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  If you are interested in being a maintainer, find ways in which you can help maintainers with their tasks;  
+  If you see an opportunity to improve the performance of our code, publish an issue suggesting a spike or changes
+
+- 🙋 **Ask** for any help needed.  
+  The codebase on Kindly is the result of a large community collaborating over time, so if there's something you're not sure about it's very likely that someone else will have an insight that can help.  
+  Be active in our discord server, share updates in your issues and if you would like to contribute in different ways let the core team know about your ideas
+
+## Collaborator
+
+Everyone involved in maintaining / improving Kindly is considered a collaborator. All you need in order to be a collaborator is the willpower to become involved.
```
Comment (eliazzo) [inline .github/GOVERNANCE.md:21]: maybe replace "All you need in order to be a collaborator is the willpower to become involved." 

with "All you need to become a collaborator is to get involved!"

**Review by eliazzo — APPROVED** [review]
I have left a couple of notes but it looks good!

File: .github/GOVERNANCE.md (hunk 1 above)
Comment (camelPhonso) [reply .github/GOVERNANCE.md:12]: I think it was but reading back this is weird so I've changed it a bit 😅

File: .github/GOVERNANCE.md (hunk 2 above)
Comment (camelPhonso) [reply .github/GOVERNANCE.md:21]: yep, definitely

**Review by eliazzo — APPROVED** [review]

**Pull request merged**